tests/test_list.py

```diff
@@ -91,6 +82,7 @@ def test_list_multiple_api_calls(client, response):
     customers = client.customers.list(limit=5)
     assert customers.count == 5
     all_customers_count = customers.count
```

Comment:
Replace the customer count with logic that collects all unique ids and counts them. See `assert_list_object()` for an example on how to do this.